packages/playwright/src/reporters/base.ts

```diff
@@ -659,3 +673,50 @@ export function resolveOutputFile(reporterName: string, options: {
 
   return { outputFile, outputDir };
 }
+
+export function fixTestPrompt(error: TestError, testFile: { path: string, contents: string }, diff?: string, pageSnapshot?: string) {
```

Comment:
Let's call it `compilePromptForError`. Also move it into a separate file.

Reply:
done